server/service/schedule/schedule.go

```diff
@@ -231,6 +231,10 @@ func (s *Schedule) Start() {
 					sentry.CaptureException(err)
 					continue
 				}
+				if newInterval <= 0 {
```

Comment:
What drove you to add this?

Reply:
I realized that the default value of `time.Duration` would be zero and thought it was a good idea to guard against it just in case.